public/js/utils/debug.js

```diff
@@ -1,11 +1,10 @@
 const { isDevelopment } = require("../feature");
+const flag = require("../test/test-flag");
```

Comment:
What do you mean? This is not a new system. This is hooking into Mozilla's existing mochitest system which sets a flag to indicate that it's in a testing environment.